sdk/eventgrid/Azure.Messaging.EventGrid/api/Azure.Messaging.EventGrid.netstandard2.0.cs

```diff
@@ -28,7 +28,7 @@ public partial class CloudEvent
     }
     public partial class EventGridEvent
     {
-        public EventGridEvent(object data, string subject, string eventType, string dataVersion, System.Type dataSerializationType = null) { }
+        public EventGridEvent(string subject, string eventType, string dataVersion, object data, System.Type dataSerializationType = null) { }
```

Comment:
moved data to be the last required param to be consistent with CloudEvent and to make it easier if inlining a new data object. The one difference is that in CloudEvent data is the first of the data-related params because the other params are optional. In EGEvent dataVersion is required.

Reply:
I still like having it last in the list of required parameters for ergonomics, even though there's a slight inconsistency between Cloud Events and Event Grid Events